src/test/java/net/snowflake/client/jdbc/SnowflakeDriverLatestIT.java

```diff
@@ -366,10 +370,13 @@ public void testPutOverwriteFalseNoDigest() throws Throwable {
     String destFolderCanonicalPath = destFolder.getCanonicalPath();
     String destFolderCanonicalPathWithSeparator = destFolderCanonicalPath + File.separator;
 
+    Properties paramProperties = new Properties();
+    paramProperties.put("GCS_USE_DOWNSCOPED_CREDENTIAL", true);
+
```

Comment:
you modified an existing test, I think we to create a new one instead of modifying it

Reply:
Based on my test, this test fail because old GCP presigned url do not support overwrite. Essentially, the parameter make it pass the test now since the new downscoped credential support list/overwrite.

@sfc-gh-stakeda Need your confirmation on this.

Reply:
Yes, that's probably true. I'm not sure what set of tests are run automatically, but this probably would have failed if there was a file that was being overwritten.

Reply:
I'm not quite understanding why this test would fail? In this test, overwrite is set to false in the PUT command, so no files are getting overwritten anyway. The command `overwrite=false` should still prevent overwrites from happening, regardless of whether the downscoped credential or the presigned url is used, right?

Reply:
No, presigned url does not support overwrite = false, so in the past, this has always been failing for GCP.

Reply:
So when using presigned url, the files are always being overwritten regardless of the OVERWRITE flag. If you check current Snowflake documentation for PUT, you will see that OVERWRITE is not supported for GCS, and this is the reason why.

Reply:
Ah I see, I thought it was the other way around (never get overwritten). Makes more sense now, thanks!